frontend/src/component/common/NewConstraintAccordion/ConstraintAccordionView/ConstraintAccordionView.tsx

```diff
@@ -95,6 +95,7 @@ export const ConstraintAccordionView = ({
                         cursor: expandable ? 'pointer' : 'default!important',
                     },
                 }}
+                tabIndex={expandable ? 0 : -1}
```

Comment:
This created an invisible tab stop on constraints that you can't expand or do anything with. Now, there is still no way to tell that your focus is on an expandable constraint accordion, but I'll look at that in a separate PR.